src/utils/getSignedData.js

```
@@ -0,0 +1,56 @@
+            if (!objSignedMessage.authors || objSignedMessage.authors.length === 0) {
+                return reject({ error: 'Validation failed' });
+            }
+
+            const { signed_message, authors: [{ address: senderWalletAddress }] } = objSignedMessage;
+
+            try {
+                const signedData = JSON.parse(signed_message.trim());
+                const { message, ...data } = signedData;
+                let attestationWalletAddress = data.address;
+
+                if (message && message.includes('I own the address:')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)